drivers/wifi/esp32/Kconfig.esp32

```diff
@@ -35,6 +35,9 @@ choice ESP_WIFI_HEAP
 	config ESP_WIFI_HEAP_SYSTEM
 		bool "Wi-Fi adapter use kernel mempool heap (k_malloc)"
 
+	config ESP_WIFI_HEAP_SPIRAM
```

Comment:
should not be indented again (lines 35-39)

Reply:
Yeah.. I followed the lines 137-140 and 206-209, that's why the indentation. Do you want me to update it all?

Reply:
sure